src/driver/communication_interface.cc

```diff
@@ -241,10 +241,18 @@ void CommunicationInterface::PublishRobotStatus() {
     franka::RobotMode current_mode {robot_data_.robot_state.robot_mode};
     robot_data_.has_robot_data = false;
     lock.unlock();
-
+    // dexai::log()->info("current mode: {} ",
+    // utils::RobotModeToString(robot_data_.robot_state.robot_mode));
     lcm_.publish(params_.lcm_status_channel, &franka_status);
+
     PublishBoolToChannel(franka_status.utime, lcm_user_stop_channel_,
                          current_mode == franka::RobotMode::kUserStopped);
+
```

Comment:
remove debugging comments and whitespace changes unless introduced by clang-format